torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1226,9 +1322,50 @@ Error PyTorchModelLoader::loadQuantizedLinear(const torch::jit::Node *ptNode) {
                                           {input.dims()[0], weight.dims()[1]},
                                           outScale, outZeroPoint - OFFSETSHIFT);
 
+  auto fc = F_.createFullyConnected("quantized_fc", input, weight, bias, outTy);
+
+  return addValueMapping(outputs[0], rescaleIntToUint(fc->getResult()));
+}
+
+Error PyTorchModelLoader::loadQuantizedLinearUnpacked(
+    const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 5, outputs, 1));
+
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      input,
+      getGlowNodeValueForValue(inputs[QuantizedUnpackedLinearInputs::input]));
+  input = rescaleUIntToInt(input);
+
+  glow::NodeValue weight;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      weight,
+      getGlowNodeValueForValue(inputs[QuantizedUnpackedLinearInputs::weight]));
+  weight = rescaleUIntToInt(weight);
```

Comment:
is this needed? Aren't linear weights stored as int8?

Reply:
Yes it is, but in case it is stored as uint8 somewhere in other models... This wont cause any trouble right